src/lib/AnnotationControls.ts

```diff
@@ -32,8 +35,47 @@ export default class AnnotationControls {
         }
 
         this.controls = controls;
+
+        this.attachEventHandlers();
+    }
+
+    /**
+     * [destructor]
+     *
+     * @return {void}
+     */
+    public destroy(): void {
+        if (fullscreen) {
+            fullscreen.removeAllListeners();
```

Comment:
This will destroy all of the listeners on the fullscreen module, right? I don't think we'll want to do that when these controls are destroyed.

Reply:
> This will destroy all of the listeners on the fullscreen module, right? I don't think we'll want to do that when these controls are destroyed.

We destroy controls when we destroy viewer, right? I think when we construct another viewer, we will create another controls and attach the fullscreen module again. So I think it's ok to remove all the listeners when controls are destroyed.

Reply:
I agree that's how it works today in practice, so it has the desired effect right now. What if, in six months, we need add logic to remove and destroy just these controls? Fullscreen would stop working and likely cause a production issue.

Encapsulation is important to reduce side effects in our code. Let's have the component remove its own handlers only.